helm/aws/basic-eks/templates/config-env.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ .Values.applicationName }}-configmap
+  namespace: {{ .Values.applicationName }}
+  labels:
+    projectName: {{ .Values.applicationName }}
+    environment: {{ .Values.environment }}
+    type: configmap
+data:
+  {{- range $key,$val := .Values.envs }}
+    {{$key}}: "{{$val}}"
```

Comment:
should we be replacing the quotes here as well with inbuild quote filter?

Reply:
yes! good find 👍

Reply:
done